### ISSUE TITLE
Compatibility with pane-relief plugin

### ISSUE THREAD

**Issue by ronrdev** [kometenstaub/customizable-page-header-buttons#25]
My main purpose for trying this plugin was to add forward/back buttons to my page header, and use a snippet to hide them on the title bar or the title bar itself. I also like to use F11 to put obsidian in full screen mode, hiding the title-bar.

Customizable-page-header-buttons plugin works as described—adding the forward/back buttons. The individual panes have their own history from the pane-relief plugin, and the pane-relief hotkeys work. 

**The right click functionality of the pain-relief plugin does not exist on the buttons added to the page header with customizable-page-header-buttons plugin.**

The right click function is basically a drop-down list of the button history to select a deeper level of navigation.

Is it possible to detect pane-relief plugin and if so replicate/use button code with the right click functions. Maybe add a toggle to turn off the numbers being displayed in-case someone would rather not have the extra visual clutter. I have low opacity on my page headers until hover so it would not bother me.

Thank you for this plugin. As it works now, I think it will be an upgrade to my workflow and anyone who uses work-spaces with multiple panes, as it makes sense to have this navigation at the page/pane level.

![Screenshot_2022-03-27_01-10-58](https://user-images.githubusercontent.com/13022919/160267702-172640f3-6e3f-4a0d-a43e-3229edbb3bda.png)

![Screenshot_2022-03-27_01-10-05](https://user-images.githubusercontent.com/13022919/160267706-c0d4427d-922e-4160-8136-73dec30451b2.png)

![Screenshot_2022-03-27_01-06-40](https://user-images.githubusercontent.com/13022919/160267733-d0531366-2bf2-4e02-92a9-ffa11585a652.png)

**Comment by kometenstaub** [comment]
That should work, pjeby added support for it three hours ago: https://github.com/pjeby/pane-relief/commit/363b7d2d64907afd5b87858f06ae2d489af3a451

**Comment by kometenstaub** [comment]
Here's CSS with which the count can be added. The alignment could be a bit better, but for that I'd need to make changes.

```css

.view-action.page-header-button.app\:go-back::before {
    content: var(--pane-relief-backward-count);
    padding-right: 0.4em;
}

.view-action.page-header-button.app\:go-forward::after {
    content: var(--pane-relief-forward-count);
    padding-left: 0.4em;
}
```

**Comment by kometenstaub** [comment]
The alignment is complicated, and needs individual treatment for themes.

```css
.view-action.page-header-button.app\:go-back::before {
    content: var(--pane-relief-backward-count);
    padding-right: 0.4em;
    font-size: 0.95em;
    vertical-align: top;
}

.view-action.page-header-button.app\:go-forward::after {
    content: var(--pane-relief-forward-count);
    padding-left: 0.4em;
    font-size: 0.95em;
    vertical-align: top;
}
```

This moves the icons down (mis-align them), but then it looks better. It needs adjustment for themes.

```css
body:not(.no-svg-replace) svg.left-arrow-with-tail {
    margin-bottom: -0.2em;
}
body:not(.no-svg-replace) svg.right-arrow-with-tail {
    margin-bottom: -0.2em;
}
```

**Comment by kometenstaub** [comment]
An updated snippet is available here: https://github.com/kometenstaub/customizable-page-header-buttons#css-snippet-for-pane-relief-count

**Comment by kometenstaub** [comment]
The count is available in the settings now as an option.

**Comment by ronrdev** [comment]
Thanks. It's working great.

**Comment by ronrdev** [comment]
Again thank you.

Adding a bug report for your reference. Not another enhancement request.

Not working as expected:
Must have the cursor in the pane if you want to use left click. Left click targets the active pane. So example if you have three panes open, all the forward buttons will change the active pane.

From pjeby
Yes, that's a limitation of the way the other plugin works, as the buttons are triggering the keyboard commands for navigating forward and back, which apply to the currently-active pane.

**Comment by kometenstaub** [comment]
Did he say how that could be fixed?

The only thing I can think of would be a debounce in these two cases, so that the click changes the active pane and then executes the command.

**Comment by ronrdev** [comment]
No - here is where I posted.
https://github.com/pjeby/pane-relief/issues/3

I added the bug report over there first because I assumed it was because of the way pane relief worked.

**Comment by kometenstaub** [comment]
That should work properly now. Please let me know whether 4.1.1 fixes it for you.

**Comment by ronrdev** [comment]
Success. Works exactly as expected and it's fast. When you mentioned setTimeout I thought it might cause a noticeable delay between button click and response, but the function is smooth. It feels like it's build in.
Thanks for creating this and adding full compatibility with pane relief. 

I'm using minimal theme, no problems.
I only tried it on desktop. I don't use mobile, but probably not a lot of multiple panes use on mobile anyway.
Only thing I had to do was copy your css from the plugin install to make a snippet to lower the opacity.
